eos-update-flatpak-repos

```diff
@@ -153,6 +154,46 @@ def _remove_runtimes():
                                '--runtime', refspec])
 
 
+# From 3.7.6 through 3.7.8, images were shipped with remote URLs using
+# our internal ostree.endlessm-sf.com domain instead of our
+# ostree.endlessm.com CDN domain.
```

Comment:
Perhaps a reference to the ticket would be good?

Reply:
Added